README.md

```diff
@@ -130,3 +130,8 @@ For most assets I use Adobe Photoshop, you can find them <a href="//github.com/t
 All assets will be automatically rebuilt into the atlas once changed (Thanks to dengr1065!)
 
 <img src="https://i.imgur.com/W25Fkl0.png" alt="shapez.io Screenshot">
+
+For generating docs:
+npm run esdoc
+Replace name.replace('\*', '\\\\\*') with name.replace('\*', '\\\\\*').replace('[', '\\\\[').replace(']', '\\\\]').replace('(', '\\\\(').replace(')', '\\\\)') in node_modules\esdoc-publish-html-plugin\out\src\Builder\DocBuilder.js:116:20)
```

Comment:
Maybe this could be formatted a bit more?

Reply:
It has loads of slashes because I need to escape the *****, **_**  and **\**. This is not shown on the github page. I now also changed it to a heading and steps

Reply:
Why don't you use code formatting?

Reply:
What do you mean?

Reply:
https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Done that. Forgot readme has that